features/projects/components/project-list/project-list.module.scss

```diff
@@ -37,3 +37,37 @@
 .loading {
   animation: spin 2s linear infinite;
 }
+
+.errorContainer {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px;
+  border-radius: 8px;
+  border: 1px solid var(--error-300, #fda29b);
```

Comment:
Instead of using CSS variables here (I'm not even sure if they work tbh) check how to use the global SCSS variables. You can see an example in line 7 where a global space variable is used. You'd need to add an import statement for the colors similar to the space at the top of the file as well.